docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,28 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed size tensor
+=================
+
+* Extension name: `arrow.fixed_size_tensor`.
+
+* The storage type of the extension: ``List``.
+
+* Extension type parameters:
+
+  * **value_type** = pyarrow DataType of the tensor elements
+  * **shape** = shape of the contained tensors as a tuple
+  * **order** = string indicating the order of elements in memory;
+    ‘C’ for row major order and ‘F’ for column major order
```

Comment:
Perhaps we should store this as `is_row_major` boolean instead?

Reply:
I have also first thought about storing this as `is_row_major` boolean but then decided to use numpy definition. Am OK with both.

Reply:
We use currently use [row_major](https://github.com/apache/arrow/blob/4acf7c17689c926841a5a471828970eda2f496c5/cpp/src/arrow/tensor.cc#L263) in Arrow, but don't use `order`. I just slightly `is_prefer row_major`.

Reply:
@jorisvandenbossche any thoughts on `is_row_major` vs `order`?